StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/New Payment Method Screen/WalletHeaderView.swift

```diff
@@ -152,14 +163,37 @@ extension PaymentSheetViewController {
 
             if let lastButton = buttons.last {
                 stackView.setCustomSpacing(Constants.labelSpacing, after: lastButton)
+                isApplePayLastButton = lastButton == applePayButton
             }
 
             addAndPinSubview(stackView)
         }
+        private func updateStackView() {
+            // Update Apple Pay Button's color
```

Comment:
Would be nice to comment why we do this - there's no way to change the color (`PKPaymentButtonStyle`) of the button, so we have to swap between light and dark versions.